master/internal/command/command.go

```diff
@@ -64,11 +64,12 @@ func createGenericCommandActor(
 	a, _ := ctx.ActorOf(cmd.taskID, cmd)
 	summaryFut := ctx.Ask(a, getSummary{})
 	if err := summaryFut.Error(); err != nil {
-		ctx.Respond(errors.Wrap(err, "failed to create generic command"))
-		return nil
+		return errors.Wrap(err, "failed to create generic command")
```

Comment:
whats the difference between ctx.Respond and returning an error here?

Reply:
we just return the err and ctx.Respond in the caller. i like to keep ctx.Respond's in the top level to keep the code as non-actory as possible